src/utils/index.ts

```diff
@@ -1,4 +1,6 @@
+import { isEmpty } from 'lodash';
 import { POPULAR_LANGUAGES } from '../constants';
+import { nameToEmoji } from 'gemoji';
```

Comment:
1. Please organize those imports
2. Check other files

Reply:
Which rule you follow in consider of imports orders?

Reply:
Ordered imports with path alphabetically

Reply:
Hmm... VS Code has the "Organize Imports" feature.

<img width="712" alt="image" src="https://github.com/AttackOnMorty/github-ranking/assets/37203901/6aa9c827-aa68-4ab0-b0c3-1413e04f1bee">